packages/amplify-provider-awscloudformation/src/root-stack-builder/root-stack-transform.ts

```diff
@@ -0,0 +1,264 @@
+import { AmplifyRootStackTemplate, Template } from './types';
+import { JSONUtilities, pathManager } from 'amplify-cli-core';
+import { AmplifyRootStack, AmplifyRootStackOutputs } from './root-stack-builder';
+import { RootStackSythesizer } from './stackSynthesizer';
+import { App } from '@aws-cdk/core';
+import * as cdk from '@aws-cdk/core';
+import { buildOverrideDir } from '../utils/override-skeleton-generator';
+
+export enum CommandType {
```

Comment:
Is this the amplify command? Could you please rename this with more information (AmplifyCLICommandType or AmplifyCommandType).